<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizItem</class>
 <widget class="QWidget" name="QuizItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_lblQuestionNumber">
     <property name="text">
      <string>Question number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_lblQuestionText">
     <property name="text">
      <string>Question text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_rbAnswerOne">
     <property name="text">
      <string>Answer one</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_rbAnswerTwo">
     <property name="text">
      <string>Answer two</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_rbAnswerThree">
     <property name="text">
      <string>Answer three</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="m_rbAnswerFour">
     <property name="text">
      <string>Answer four</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="m_lblWarning">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_btnNext">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnConfirm">
       <property name="text">
        <string>&amp;Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
